runelite-client/src/main/java/net/runelite/client/game/ItemManager.java

```diff
@@ -460,6 +460,23 @@ public AsyncBufferedImage getImage(int itemId)
 	 * @param quantity
 	 * @return
 	 */
+	public AsyncBufferedImage getImage(int itemId, int quantity)
```

Comment:
Why is this added? It's not used...

Reply:
I had added it mostly to keep in line w/ how I've always done overloaded methods in Java (which is have the least qualified method call increasingly verbose methods w/ defaults) but there is no actual reason for it so I've removed this method overload.